projects/storefrontlib/shared/models/file.ts

```diff
@@ -0,0 +1,21 @@
+export interface FileOptions {
+  /**
+   * File name for exported file.
+   */
+  fileName: string;
+  /**
+   * Extension for exported file.
+   */
+  extension: string;
+  /**
+   * Mime/type for exported file.
+   */
+  type: string;
+}
+
+export interface InvalidFileInfo {
+  tooLarge?: { maxSize: number };
+  tooManyEntries?: { maxEntries: number };
+  empty?: boolean;
+  notParsable?: boolean;
+}
```

Comment:
Since those guys live in the `@spartacus/storefront`, they should have more specific names (belong to features which use them) and moved to those feature's folders.

Otherwise we will have very specific opinionated interfaces with very generic names imported from `@spartacus/storefront`.

Reply:
renamed to `ExportFileOptions` and `CsvFileValidationErrors`